src/emit/solana.rs

```diff
@@ -2963,6 +2982,66 @@ impl<'a> TargetRuntime<'a> for SolanaTarget {
         }
     }
 
+    fn builtin_function(
+        &self,
+        binary: &Binary<'a>,
+        func: &ast::Function,
+        args: &[BasicMetadataValueEnum<'a>],
+        ns: &ast::Namespace,
+    ) -> BasicValueEnum<'a> {
+        assert_eq!(func.name, "create_program_address");
```

Comment:
If you are restricting the function that should be use with this method, its name should say so.